questions.json

```
@@ -2038,7 +2038,7 @@
       "id": 516,
       "question": {
         "en": "What would you be doing right now if we weren\u2019t having this meeting? How do you feel about being taken away from that task?",
-        "ja_JP": "もしこの打ち合わせがなかったら何していましたか？ やろうとしてなかったことをできなかったことについてどう感じていますか？"
+        "ja_JP": "もしこの打ち合わせがなかったら何していましたか？ その仕事から連れ去られてここにいることについてどう感じていますか？"
```

確かに、ちょっと踏み込んだ意訳気味ではありますね。
ただ、直訳の「連れ去る」だとちょっと詩的過ぎる感もあります。
こんな訳ならどうでしょう？

```suggestion
        "ja_JP": "もしこの打ち合わせがなかったら何していましたか？ そのタスクから引き離されたことについてどう感じていますか？"
```